src/targets/cpu/include/migraphx/cpu/dnnl.hpp

```diff
@@ -91,6 +91,19 @@ struct post_op : reflect_equality<post_op>, reflect_stream<post_op>
     }
 };
 
+template <class F>
```

Comment:
Can you add a comment about wrapping this lambda because an issue with MSVC and the `context` parameter?